app/src/main/java/de/danoeh/antennapod/fragment/SearchFragment.java

```diff
@@ -133,6 +140,13 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         emptyViewHandler.setIcon(R.attr.action_search);
         emptyViewHandler.setTitle(R.string.search_status_no_results);
         EventBus.getDefault().register(this);
+
+        chip = layout.findViewById(R.id.feed_title_chip);
+        if ((getArguments().getLong(ARG_FEED) != 0) && !FEED_TITLE.isEmpty()) {
+            chip.setText(FEED_TITLE);
+        } else {
+            chip.setVisibility(View.GONE);
+        }
```

Comment:
Please move this to the results section of the `search` method. This makes it possible to add a close button to the chip (which would greatly improve the user experience) with something like:

```
        app:closeIconVisible="true"
```

```
        chip.setOnCloseIconClickListener(v -> {
            getArguments().putLong(ARG_FEED, 0);
            search();
        });
```

Reply:
Wow, really liked the feature. Wouldn't have thought about it 

Adding  it too.